### PR TITLE
2

### PR DESCRIPTION
**Pull request by marcbloc**

### CODE REVIEW

File: js/Column.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
+function Column(id, name) {
+  var self = this;
+  
+  this.id = id;
+  this.name = name || 'No name given';
+  this.element = createColumn();
+
+  function createColumn() {
+    // TWORZENIE NOWYCH WĘZŁÓW
```
Comment (matwjablonski) [inline js/Column.js]: don't use polish in code

File: js/Column.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    var column = $('<div class="column"></div>');
+    var columnTitle = $('<h2 class="column-title">' + self.name + '</h2>');
+    var columnCardList = $('<ul class="card-list"></ul>');
+    var columnDelete = $('<button class="btn-delete">x</button>');
+    var columnAddCard = $('<button class="column-add-card">Dodaj kartę</button>');
+    
+    // PODPINANIE ODPOWIEDNICH ZDARZEŃ POD WĘZŁY
```
Comment (matwjablonski) [inline js/Column.js]: don't use polish in code

File: index.html [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+	<div id="board" class="board">
+	  <h1>Tablica kanban</h1>
+	  <button class="create-column">Dodaj kolumnę</button>
+	  <div class="column-container"></div>
+	</div>
+
+    <script src="https://code.jquery.com/jquery-3.0.0.min.js" integrity="sha256-JmvOoLtYsmqlsWxa7mDSLMwa6dZ9rrIdtrrVYRnDRH0=" crossorigin="anonymous"></script>
+    <script   src="https://code.jquery.com/ui/1.11.4/jquery-ui.min.js"   integrity="sha256-xNjb53/rY+WmG+4L6tTl9m6PpqknWZvRt0rO1SRnJzw="   crossorigin="anonymous"></script>
+    <!--TO JEST JUŻ NIE POTRZEBNE-->
```
Comment (matwjablonski) [inline index.html]: if it's unnecesary just remove it :)

File: index.html [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+    <!--<script type="text/javascript" src="script.js"></script>-->
+    
+    <!--TERAZ UŻYWAMY TEGO-->
+    <script type="text/javascript" src="Column.js"></script>
```
Comment (matwjablonski) [inline index.html]: wrong src

File: index.html [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+    <script type="text/javascript" src="Card.js"></script>
```
Comment (matwjablonski) [inline index.html]: wrong src

File: index.html [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+    <script type="text/javascript" src="Board.js"></script>
```
Comment (matwjablonski) [inline index.html]: wrong src

File: index.html [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+    <script type="text/javascript" src="App.js"></script>
```
Comment (matwjablonski) [inline index.html]: wrong src

File: js/App.js [hunk 8]
```diff
@@ -0,0 +1,30 @@
+var baseUrl = 'https://kodilla.com/pl/bootcamp-api';
+var myHeaders = {
+	'X-Client-Id': '1834',
+	'X-Auth-Token': '9f48b53d7e4c2f76e73f29a8cd27519b'
+};
+
+$.ajaxSetup({
+	headers: myHeaders;
```
Comment (matwjablonski) [inline js/App.js]: remove `;`

File: index.html [hunk 9]
```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8">
+    <title>Tablica kanban</title>
+    <link rel="stylesheet" href="style.css">
```
Comment (matwjablonski) [inline index.html]: wrong src

File: js/Column.js [hunk 10]
```diff
@@ -0,0 +1,64 @@
...
+        url: baseUrl + '/card',
+        method: 'POST',
+        data: {
+          name: cardName,
+          bootcamp_kanban_column_id: self.id
+        },
+        success: function(response) {
+          var card = new Card(response.id, cardName);
+          self.createCard(card);
+          console.log(response)
```
Comment (matwjablonski) [inline js/Column.js]: remove console.log

File: js/Column.js [hunk 11]
```diff
@@ -0,0 +1,64 @@
...
+      });
+    });
+
+// KONSTRUOWANIE ELEMENTU KOLUMNY
+      column.append(columnTitle)
+      .append(columnDelete)
+      .append(columnAddCard)
+      .append(columnCardList);
+      return column;
+
```
Comment (matwjablonski) [inline js/Column.js]: too many empty lines

File: js/script.js [hunk 12]
```diff
@@ -0,0 +1,121 @@
+$(function(){
+
+  // FUNKCJE POMOCNICZE
```
Comment (matwjablonski) [inline js/script.js:3]: in english

**Review by matwjablonski — CHANGES_REQUESTED** [review]

**Comment by marcbloc** [comment]
corrected

**Comment by marcbloc** [comment]
corrected. please check

**Review by matwjablonski — APPROVED** [review]

**Pull request merged**